src/js/server/utils.js

```diff
@@ -49,14 +49,30 @@ fluid.defaults("fluid.express.user.utils", {
     codeKey:          "verification_code",  // Must match the value in fluid.express.user.verify
     rules: {
         createUserWrite: {
-            "name":          "username", // Default rules are designed to cater to CouchDB  and express-couchUser conventions, but can be overriden.
-            "username":      "username",
-            "email":         "email",
-            roles:           { literalValue: []},
-            type:            { literalValue: "user"},
-            password_scheme: { literalValue: "pbkdf2"},
-            iterations:      { literalValue: 10},
-            verified:        { literalValue: false}
+            // TODO: Discuss w/ Antranig why this isn't equivalent to:
```

Comment:
Transforms don't support expansion in the members "inputPath" and "outputPath" - these are always interpreted as literal paths or path segments

Reply:
Thanks.